Backpack/src/main/res/anim/dialog_slide_out_down.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<translate
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  android:duration="200"
```

Comment:
we should consider having animation duration tokens some time soon. @willhg

Reply:
The animation is again based on the IOS version here and is trying to copy that. I checked the animation with @willhg (and the component in general). I agree tokens would be good but they should not be part of this PR, and also the IOS version does the same so we would need to update both.

Reply:
@willhg agreed to look into this and we will take it up later